enhancements/kube-apiserver/bound-sa-tokens.md

```diff
@@ -69,17 +82,21 @@ therefore risk of compromise) for a given service account token.
     in kube > 1.12, but are only configured by the apiserver if the following options are
     provided:
     - service-account-signing-key-file
-      - Operator should set this to the name of a private key it manages
+      - Operator should set this to the path of the private key it manages
     - service-account-issuer
-      - Operator should set this to https://kubernetes.default.svc/
+      - Operator should default this to `bound-sa-token.auth.openshift.io` and it should be
```

Comment:
if this is changed, which consequences does it have (especially for existing tokens) ?

Reply:
Existing tokens signed by the previous issuer will fail to validate. If long-lived tokens have been issued to pods, pod restart will be the only fix.